src/components/RecommendedFile.vue

```diff
@@ -78,6 +79,21 @@ export default {
 			type: Boolean,
 			default: false,
 		},
+		timestamp: {
+			type: Number,
+			default: null,
+		},
+	},
+	setup(props) {
+		if (!props.timestamp) {
+			return
+		}
+		const { formattedTime } = useFormatDateTime(props.timestamp * 1000, {
+			ignoreSeconds: true,
+		})
+		return {
+			formattedTime,
+		}
```

Comment:
This is not how setup context work. Setup is executed *once* before the mount / after creation of the component. So if the props are changed you do not get the formatted time.

So if timestamp is required it should look like:

```suggestion
	setup(props) {
		const { formattedTime } = useFormatDateTime(computed(() => props.timestamp * 1000), {
			ignoreSeconds: true,
		})
		return {
			formattedTime,
		}
```